### ISSUE TITLE
Each db migration causes rubocop errors

### ISSUE THREAD

**Issue by danstewart** [Williams-Dan/covidhelp#4]
Each time `rake db:migrate` is ran it rewrites `schema.rb` and adds lines longer than 80 chars which rubocop isn't liking.

```
/code/covidhelp/vendor/ruby/2.7.0/gems/activesupport-4.2.11.1/lib/active_support/core_ext/object/duplicable.rb:111: warning: BigDecimal.new is deprecated; use BigDecimal() method instead.
gem list '^rubocop$' -i
true
rubocop -a
Inspecting 14 files
...........C..

Offenses:

db/schema.rb:5:81: C: Layout/LineLength: Line is too long. [81/80]
# incrementally modify your database, and then regenerate this schema definition.
                                                                                ^
db/schema.rb:10:81: C: Layout/LineLength: Line is too long. [86/80]
# from scratch. The latter is a flawed and unsustainable approach (the more migrations
                                                                                ^^^^^^
db/schema.rb:13:81: C: Layout/LineLength: Line is too long. [86/80]
# It's strongly recommended that you check this file into your version control system.
                                                                                ^^^^^^

14 files inspected, 3 offenses detected
rake aborted!
Command failed with status (1): [rubocop -a...]
/code/covidhelp/Rakefile:25:in `block in <top (required)>'
Tasks: TOP => run => lint
(See full trace by running task with --trace)
```

**Comment by ravxl** [comment]
We can create a `.rubocop.yml` to deal with it. It's possible to ignore this file or use a longer limit to lines.

```yaml
Metrics/LineLength:
  # This will disable the rule completely, regardless what other options you put
  Enabled: false
  # Change the default 80 chars limit value
  Max: 120
  # If you want the rule only apply to a specific folder/file
  Include:
    - 'app/**/*'
  # If you want the rule not to apply to a specific folder/file
  Exclude:
    - 'db/schema.rb'
```

**Comment by ravxl** [comment]
Fixed on #7
